ocfweb/tv/templates/tv/tv-base.html

```diff
@@ -0,0 +1,30 @@
+{% load staticfiles %}
```

Comment:
Yeah I was trying to fix the the app prefix error and just moved the template in, I'll rename this